package.json

```diff
@@ -15,6 +17,7 @@
     "eslint-config-prettier": "^8.3.0",
     "husky": "^7.0.2",
     "lint-staged": "^11.2.3",
+    "next": "^11.1.2",
```

Comment:
Is this necessary? I don't think we use next outside of the nextjs examples.

Reply:
Basically I think we should run lint per sample (concurrently with lerna)